emsdk.py

```diff
@@ -665,6 +665,11 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix='
         sys.stdout.flush()
   except Exception as e:
     print("Error downloading URL '" + url + "': " + str(e))
+    # Modified code to inform solution for common "...[SSL: CERTIFICATE_VERIFY_FAILED] ..." on Mac and
```

Comment:
Can remove "modified code", as that makes more sense in a comment message, I think, than the text that remains in the code forever. It could start with "Hint at a solution for the common"

Reply:
I think this has not yet been addressed?

Reply:
> Can remove "modified code", as that makes more sense in a comment message, I think, than the text that remains in the code forever. It could start with "Hint at a solution for the common"

Removed in the latest pull request.